runtime/android/core_internal/src/org/xwalk/core/internal/XWalkMediaPlayerResourceLoadingFilter.java

```diff
@@ -39,10 +47,20 @@ public boolean shouldOverrideResourceLoading(MediaPlayer mediaPlayer,
                     context.getAssets().openFd(AndroidProtocolHandler.getAssetPath(uri));
             mediaPlayer.setDataSource(
                     afd.getFileDescriptor(), afd.getStartOffset(), afd.getLength());
-
             return true;
         } catch (Exception e) {
             return false;
         }
     }
+
+    @Override
+    public ExMediaPlayer getExMediaPlayer() {
+        ExMediaPlayer exMediaPlayer = mContentsClientBridge.getExMediaPlayer();
+
+        if (exMediaPlayer == null) {
+            exMediaPlayer = new ExMediaPlayer();
+        }
```

Comment:
Shouldn't these code be implemented within `mContentsClientBridge.getExMediaPlayer()`?